kong/db/strategies/postgres/connector.lua

```diff
@@ -329,23 +329,22 @@ function _mt:init_worker(strategies)
       }
     end
 
-    cleanup_statement = concat({
-      "BEGIN;",
-      concat(cleanup_statement, "\n"),
-      "COMMIT;"
-    }, "\n")
-
     return timer_every(60, function(premature)
       if premature then
         return
       end
 
-      local ok, err = self:query(cleanup_statement)
-      if not ok then
-        if err then
-          log(WARN, "unable to clean expired rows from postgres database (", err, ")")
-        else
-          log(WARN, "unable to clean expired rows from postgres database")
+      for i, statement in ipairs(cleanup_statements) do
+        local ok, err = self:query(statement)
+        if not ok then
+          if err then
+            log(WARN, 'unable to clean expired rows from table "',
```

Comment:
style: can we stick to double quotes? We use single quotes in the text of each log line we write.